libs/domain/search/facet/facet.component.ts

```diff
@@ -35,7 +36,7 @@ export class SearchFacetComponent extends I18nMixin(LitElement) {
   }
 
   protected override render(): TemplateResult | void {
-    const facet = this.facet();
+    const facet = this.facet() as SingleMultiFacet;
 
     if (!facet) return;
```

Comment:
Why we don't check on type here. We can event change interfaces that we understod by types what in the facet.

Reply:
refactored